### ISSUE TITLE
Firefox 77.0.1 scrambles pages

### ISSUE THREAD

**Issue by accetto** [accetto/xubuntu-vnc-novnc#2]
Unfortunatelly it has been found, that **Firefox 77.0.1** in the containers from the release [20.06](https://github.com/accetto/xubuntu-vnc-novnc/releases/tag/v20.06) scrambles web pages. Therefore I'll remove all the related images and come with at least some way arround as quickly as possible. Sorry for the inconvenience.

![Screenshot_Firefox_77 0 1_scrambles_pages](https://user-images.githubusercontent.com/34798830/84268608-755f5180-ab17-11ea-92b7-656d6a218c66.png)

**Comment by accetto** [comment]
Mitigated in the release **20.06.1** by rolling **Firefox** back to the version **76.0.1**.

For the case that you get the message `You've launched an older version of Firefox` please check [Firefox version compatibility](https://github.com/accetto/xubuntu-vnc/wiki/Firefox-version-compatibility) in Wiki.

In essence it explains, that if you want to keep using your previous Firefox profile, you have to replace the content of its- **compatibility.ini** file. 

If it looks like this

```
[Compatibility]
LastVersion=77.0.1_20200602222727/20200602222727
LastOSABI=Linux_x86_64-gcc3
LastPlatformDir=/usr/lib/firefox
LastAppDir=/usr/lib/firefox/browser
```

then replace it with this:

```
[Compatibility]
LastVersion=76.0.1_20200507114007/20200507114007
LastOSABI=Linux_x86_64-gcc3
LastPlatformDir=/usr/lib/firefox
LastAppDir=/usr/lib/firefox/browser
```

You can also create and use a new profile, of course. With the **FF Profile Manager**, which you can find on the desktop, you can easily switch between several profiles.

**Comment by accetto** [comment]
This issue will further commented in [issue #7 of accetto/xubuntu-vnc](https://github.com/accetto/xubuntu-vnc/issues/7). Please check it there.

**Comment by accetto** [comment]
It has been fixed in the release 20.06.2 by making the Firefox multi-process default.
